build/rollup.config.js

```diff
@@ -89,6 +89,8 @@ export default inputs.map(input => {
                     module: Handlebars
                 },
                 helpers: ['build/tpl.js'],
+                isPartial : () => true,
+                partialRoot: srcDir,
```

Comment:
The new system works well with or without these 2 lines. This part of the helper library doesn't seem to do anything now that we're defining the partial names as module paths.